.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
The typical haskell style is to do:

```haskell
myList :: [String]
mylist = [ "foo"
         , "bar"
         , "baz"
         ]
```

I have seen this repeated in many JS projects in the past and it works for them.